libr/bin/p/bin_mach0.c

```diff
@@ -284,6 +289,12 @@ static RList* symbols(RBinFile *bf) {
 	}
 	bool isStripped = false;
 	wordsize = MACH0_(get_bits) (obj->bin_obj);
+	// NEW CODE
+	obj->symbols = ret;
+	MACH0_(pull_symbols)(obj->bin_obj, pull_symbol, obj);
+return ret;
```

Comment:
remove the old code after this `return`.
Indent this return (if you want you can try `git diff master..bin-pull | ./sys/clang-format-diff.py -p1 -i` :D )